analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -166,8 +166,8 @@ class GoMod(
             val columns = line.splitOnWhitespace()
             require(columns.size == 2) { "Expected exactly one occurrence of ' ' on any non-blank line." }
 
-            val parent = parseModuleEntry(columns[0])
-            val child = parseModuleEntry(columns[1])
+            val parent = parseModuleEntry(columns.first())
+            val child = parseModuleEntry(columns.last())
```

Comment:
I believe [1] is more speaking than `last` and more appropriate.
Due to that I believe [0] should also be kept for consistency.

Reply:
I've reverted that occurrence.